scripts/benchmarks_ci.py

```diff
@@ -250,8 +251,6 @@ def __main(args: list) -> int:
                 target_framework_moniker = micro_benchmarks.FrameworkAction.get_target_framework_moniker(framework)
                 commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli)
                 source_timestamp = dotnet.get_commit_date(framework, commit_sha, args.cli_repository)
-                os.environ['PERFLAB_HASH'] = commit_sha
-                os.environ['PERFLAB_BUILDTIMESTAMP'] = source_timestamp
 
```

Comment:
does it make sense to just roll back the entire change that brought this in at this point? (maybe it doesn't matter.)

Reply:
Oh, right, extra delete!